### ISSUE TITLE
JWT support

### ISSUE THREAD

**Issue by avara1986** [python-microservices/pyms#66]
Create a service to support JWT like https://github.com/python-microservices/oauth

**Comment by KingDarBoja** [comment]
Hello, just curious, I was checking the provided oauth microservice and I do see you guys are using `flask-jwt`. But wouldn't be better to use other library for jwt which has better maintenance and is up-to-date like as [pyjwt](https://pyjwt.readthedocs.io/en/latest/) as example?

Cheers!

**Comment by avara1986** [comment]
Hi @KingDarBoja! Thanks for your recommendation! :+1:  We create the [oauth MS] (https://github.com/python-microservices/oauth) with flask-jwt because it works with PyJWT, but we don't saw that flask-jwt has no maintenance. If someone want to create this service could be like # 62: The metrics with prometheus in python has got libraries with flask implementation with no maintenance and we create a similar implementation like these

**Comment by KingDarBoja** [comment]
I am interested on such thing but need some time to know more about this project along with its dependencies. Is there any gitter where you guys chatting and also answer question for newcomers?

**Comment by avara1986** [comment]
> Is there any gitter where you guys chatting and also answer question for newcomers?

No, but it's good idea :smile: See https://github.com/python-microservices/pyms/pull/80

**Comment by DiptoChakrabarty** [comment]
hey can we use flask-jwt or flask-jwt-extended for this service

**Comment by DiptoChakrabarty** [comment]
Hey can I work on this , I think we can use flask jwt extended which is relatively much better maintained and up-to date compared  to flask-jwt . I would like to send the PR in the oauth micro service repository

**Comment by avara1986** [comment]
It's a great Idea @DiptoChakrabarty :) Flask JWT Extended is more mantained than Flask JWT! :+1:
